clients/admin-ui/src/types/api/models/ScopeRegistryEnum.ts

```diff
@@ -60,6 +56,7 @@ export enum ScopeRegistryEnum {
   STORAGE_CREATE_OR_UPDATE = "storage:create_or_update",
   STORAGE_DELETE = "storage:delete",
   STORAGE_READ = "storage:read",
+  SYSTEM_READ = "system:read",
```

Comment:
hmm was this added recently? I remember thinking it wasn't a thing

Reply:
It's added in Dawn's BE work here- https://github.com/ethyca/fides/pull/2714/files#diff-64b8825251ee956a60b62e71f974e359f0b540694f46dd581d2b524c2a94617bR184